examples/embedding/Makefile

```diff
@@ -6,3 +6,8 @@ hello-embed: hello-embed.o -lmicropython
 
 -lmicropython:
 	$(MAKE) -f $(MPTOP)/examples/embedding/Makefile.upylib MPTOP=$(MPTOP)
+
+clean:
+	rm -rf build
```

Comment:
Normally this should be $(RM) but I'm not sure if it's ok to include mkenv.mk here?

Reply:
make sense, done